lib/config/themes.dart

```diff
@@ -72,6 +73,10 @@ abstract class FluffyThemes {
       useMaterial3: true,
       brightness: brightness,
       colorScheme: colorScheme,
+      cupertinoOverrideTheme: const CupertinoThemeData(
+        // This is required by adaptive_dialog
```

Comment:
Can you explain here, why it is required?

Reply:
The issue:
![image](https://github.com/user-attachments/assets/09cc10e3-b268-4890-b736-a975d53e1f89)

The fix:
I thought this should be a common issue, so I've searched if there are any instructions.
and...
The change is guided by the readme of `adaptive_dialog` https://pub.dev/packages/adaptive_dialog#the-input-text-color-same-with-backgound-when-using-cupertinotextinputdialog